CHANGES.md

```diff
@@ -16,25 +16,25 @@
   class.  `Integral` contains the bitvector types and `Integer`.
 
   The new `Field` class contains types representing mathematical
-  fields (or types that are approximately fields). For now, it is
-  inhabited only by the new `Rational` type, but will eventually
-  contain `Real` and floating-point types. It has the operation
-  `recip` for reciprocal and `(/.)` for field division (not to be
-  confused for `(/)`, which is Euclidean integral division).
+  fields (or types that are approximately fields). It is currently
+  inhabited by the new `Rational` type, and the `Float`
+  family of types.  It will  eventually also contain the
```

Comment:
Some flagrantly nitpicky feedback: you have a mix of single and double spaces here (`will  eventually`)